docs/conf/portland/2021/welcome-wagon.rst

```diff
@@ -259,3 +240,64 @@ Pro tips
 - Find out who is attending the conference before you attend. Join the `Write the Docs Slack <http://slack.writethedocs.org/>`__, follow the `Write the Docs on Twitter <https://twitter.com/writethedocs>`__, and review the `list of speakers <https://www.writethedocs.org/conf/{{ shortcode }}/{{ year }}/speakers/>`__.
 
 - In case of technical issues with our main platform Hopin, please watch the `#wtd-conferences Slack channel <https://writethedocs.slack.com/archives/C1AKFQATH>`_ for status updates and alternative viewing links that might be posted.
+
+FAQs
+-----------------
+
+Are the talks recorded?
+- Stage talks and Q&A sessions are recorded. Videos will be published in the ` -Write the Docs Video Archive <https://www.writethedocs.org/videos/>`__ shortly after the conference (typically about a week after the conference ends).
+- Other sessions (such as Unconference, Hallway, and Writing Day sessions) are not recorded.
+
+How do I get into the virtual conference?
+- Getting into the conference is a two-step process.
+- You buy a ticket, and then you get a link to register. During this registration process, you will get a link to get into the conference.
+
+Do I have to be on video during the conference?
+- You can choose whether to share your video during the conference, including during Unconference sessions. Select the Video Camera icon to turn on and off your video.
+- You can also share your screen during an Unconference. Select the Monitor icon to share your screen. Multiple people can share their screens at once.
+- Select the Settings icon to change your audio or video settings.
+- To make someone’s screen larger, double-click on it.
+ 
+How do I see what’s happening right now?
+- Go to Reception to see the talk schedule.
+- Go to the Write the Docs website to view the `_full schedule <https://www.writethedocs.org/conf/portland/2021/schedule/>`__.
+- View the `_Integrated schedule <https://docs.google.com/spreadsheets/d/1bE3_p0EkfPrcQzfbPj-7B0mGZQwzOo2V10jEqiVQ6hk/edit#gid=0>`__ to see what Writing Day and Unconference sessions are happening.
```

Comment:
```suggestion
- View the `_Integrated schedule <{{unconf.url}}>`__ to see what Writing Day and Unconference sessions are happening.
```
I think this one should also point to the read only one, as the text refers to "view".